packages/keystone/fields/SignedDecimal/Implementation.js

```diff
@@ -16,6 +16,7 @@ class SignedDecimal extends Decimal.implementation {
 
     async validateInput (args) {
         const { resolvedData, fieldPath, addFieldValidationError } = args
+        if (resolvedData[fieldPath] === '') resolvedData[fieldPath] = null
```

Comment:
Need a comment here, that in another case admin-ui do not allow reset field to null